README.md

```diff
@@ -25,7 +30,35 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 $ make image
 ```
 
-### Start Image driver
+### Installing into Kubernetes
+```
+deploy/deploy-image.sh
+```
+
+### Example Usage in Kubernetes
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.13-alpine
+    ports:
+    - containerPort: 80
+    volumeMount:
+    - name: data
+      mountPath: /usr/share/nginx/html
+  volumes:
+  - name: data
+    csi:
+      driver: csi-image
```

Comment:
I know we don't do this currently for the other sample drivers, but do we want to have a prefix, like "storage.sigs.k8s.io"?

Reply:
works for me. so, storage.sigs.k8s.io/image ? or image.storage.sigs.k8s.io?

Reply:
Oh actually driver name is not a prefix.  @saad-ali suggests "image.csi.k8s.io"

Reply:
Sounds good to me.